sdks/python/apache_beam/runners/worker/statecache.py

```diff
@@ -44,15 +48,17 @@ class Metrics(object):
   PREFIX = "beam:metric:statecache:"
 
   def __init__(self):
+    # type: () -> None
     self._context = threading.local()
 
   def initialize(self):
+    # type: () -> None
+
     """Needs to be called once per thread to initialize the local metrics cache.
     """
     if hasattr(self._context, 'metrics'):
       return  # Already initialized
-    self._context.metrics = collections.defaultdict(
-        int)  # type: DefaultDict[Hashable, int]
```

Comment:
Removed this annotation because it's invalid:   you can't add annotations to an attribute of another class (non-self attribute).  I've noticed there's definitely a need for a typed version of `threading.local`.